src/tests/JIT/opt/perf/doublenegate/doublenegate.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace coreclr_test_13647
+{
+    class doublenegate
+    {
+        static void Main(string[] args)
+        {
+            Test1();
+            Test2();
+        }
+
+        static void Test1()
+        {
+            var x = DoubleNegationInt(6);
+            var y = 
+              DoubleNegationInt(
+                DoubleNegationInt(
+                  DoubleNegationInt(x)));
+            Console.WriteLine(y);
+        }       
+
+        [MethodImpl(MethodImplOptions.AggressiveOptimization | MethodImplOptions.AggressiveInlining)]
+        static int DoubleNegationInt(int value) => -(-value);
```

Comment:
-(-value) can be at some point optimized by Roslyn (who knows) so I'd suggest to just use `-value` here

Reply:
Done, thanks!